### DISCUSSION TITLE
Testing with Gematik Kops connector simulator

### DISCUSSION THREAD

**Discussion by norganim** [Digitaler-Impfnachweis/certification-apis#121] [Q&A]
Hi @oliver-steinbrecher 

#14 

when trying to get a x-auth-challenge via https://id.certify.demo.ubirch.com/auth/realms/ubirch-certify/protocol/openid-connect/auth, the response is "Client not found".  

What is the correct client_id? 

Thanks & have a nice evening

**Comment by oliver-steinbrecher** [comment]
What client do you use?

**Reply by norganim** [reply]
We use client_id= smcb-auth-simulator, as written in the documentation.

**Comment by norganim** [comment]
Hi,

any new knowledge, about the problem?

**Comment by ftxxl** [comment]
We get the same problem. Curiously, we only get an x-auth-challenge with client_id=user-access-ti

**Reply by norganim** [reply]
That's correct, we got also an x-auth-challenge with user-access-ti, do you get an token?
Cause we got an error "Certificate & Signature Verification failed".

**Reply by ftxxl** [reply]
Well, same here

**Reply by norganim** [reply]
Ok, seems we have to wait for a fix or a clarification.

**Reply by nicolaslang1** [reply]
I tried every combination of realm and client_id. For example: Realm "bmg-ti-certify" with client_id "user-access-ti-kops-simulator" also works and returns a token at the end. Unfortunately the error "Invalid JWT Token" appears again.  Do you guys know if i can test in production environment? We don't have much time left to wait for bugfixxes ...

**Reply by ftxxl** [reply]
since it still doesn't work on oru behalf, i'd try a mention of @oliver-steinbrecher in this thread.
